mne/beamformer/_lcmv.py

```diff
@@ -51,7 +51,7 @@ def _reg_pinv(x, reg):
 
 
 def _setup_picks(info, forward, data_cov=None, noise_cov=None):
-    """Return good channels common to forward model and covariance matrices."""
+    """Returns good channels common to forward model and cov. matrices."""
```

Comment:
My apologies @britta-wstnr it was my bad, just getting used to linters actually dealing with PEP257 https://www.python.org/dev/peps/pep-0257/#one-line-docstrings just roll back this commit.